test/upgrade/mzcompose.py

```diff
@@ -216,6 +216,12 @@ def test_upgrade_from_version(
             f"check-{style}from-{version_glob}-{filter}.td",
         )
 
+    # Ensure we can restart after checking the upgrade tests.
```

Comment:
You noticed something off in the tests without this?

Reply:
I meant this as a light integration test for reconciliation, i.e. once we have the catalog full of objects and we reboot, does reconciliation cause anything to fall over? However, there are many restart tests so this isn't testing anything new/meaningful. Will remove.